catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -249,6 +413,177 @@ define([
                     west: this.get('mapWest')
                 });
             }
+        },
+
+        // Convert Lat-Lon to UTM coordinates. Returns undefined if lat or lon is undefined or not a number.
+        // Returns undefined if the underlying call to usng fails. Otherwise, returns an object with:
+        //
+        //   easting    : FLOAT
+        //   northing   : FLOAT
+        //   zoneNumber : INTEGER (>=1 and <= 60)
+        //   hemisphere : STRING (NORTHERN or SOUTHERN)
+        LLtoUTM: function(lat,lon) {
+
+            if(lat === undefined || lon === undefined || isNaN(lat) || isNaN(lon)) {
```

Comment:
isNaN per this should handle the undefined case here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN